projects/plugins/crm/src/js/components/automations-admin/components/attribute-config/index.tsx

```diff
@@ -61,6 +61,7 @@ const getEditValue = (
 							  } ) )
 							: []
 					}
+					value={ value }
```

Comment:
This just fixes a bug where on load the SelectControl would display the first option instead of the option selected in the store.